### PR TITLE
remove old lease impl in hub side

### PR DESCRIPTION
**Pull request by marcolan018**
Signed-off-by: Marco <llan@redhat.com>

### CODE REVIEW

File: controllers/placementrule/endpoint_metrics_operator.go [hunk 1]
```diff
@@ -102,6 +99,7 @@ func updateEndpointOperator(mco *mcov1beta2.MultiClusterObservability,
 	namespace string, container corev1.Container) corev1.Container {
 	container.Image = getImage(mco, mcoconfig.EndpointControllerImgName,
 		mcoconfig.EndpointControllerImgTagSuffix, mcoconfig.EndpointControllerKey)
+	container.Image = "blue0/endpoint-monitoring-operator:0.1.0"
```
Comment (clyang82) [inline controllers/placementrule/endpoint_metrics_operator.go:102]: This is for testing. need to remove.

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: controllers/placementrule/endpoint_metrics_operator.go (hunk 1 above)
Comment (marcolan018) [reply controllers/placementrule/endpoint_metrics_operator.go]: @clyang82 
removed

**Pull request merged**